### PR TITLE
Fix attribute handling

### PR DESCRIPTION
**Pull request by tronsha**
Two small adjustments to attribute handling.
"stripEventHandlers" works via PHPUnit, but removes only the first event without the adjustment.

### CODE REVIEW

File: tests/fixtures/vue-bind/binding-with-template-string.vue [hunk 1]
```diff
@@ -4,6 +4,9 @@
       Hello World
     </div>
     <div :class="`${isTrue ? 'a' : 'b'}`"></div>
+    <div :style="'display: none !important'">
```
Comment (Macavity) [inline tests/fixtures/vue-bind/binding-with-template-string.vue]: This is not a template string anymore, can you also test that the template string binding keeps on working?

File: src/Compiler.php [hunk 2]
```diff
@@ -579,12 +580,16 @@ private function transformCamelCaseToCSS(string $property): string
 
     private function stripEventHandlers(DOMElement $node)
     {
+        $removeAttributes = [];
         /** @var DOMAttr $attribute */
         foreach ($node->attributes as $attribute) {
             if (strpos($attribute->name, 'v-on:') === 0 || strpos($attribute->name, '@') === 0) {
-                $node->removeAttribute($attribute->name);
```
Comment (gulnurullah) [inline src/Compiler.php]: Is not much faster do here removeAttribute rather than put another loop ?

Comment (tronsha) [reply src/Compiler.php]: The old solution is not working properly in our project. Only the first attribute was removed. Unfortunately I couldn't find out why, so I had this solution, which first collects all attributes and then removes them later.

File: tests/fixtures/vue-bind/binding-with-template-string.vue (hunk 1 above)
Comment (tronsha) [reply tests/fixtures/vue-bind/binding-with-template-string.vue]: I moved it to binding.vue

File: src/Compiler.php (hunk 2 above)
Comment (tronsha) [reply src/Compiler.php]: The old solution is not working properly in our project. Only the first attribute was removed. Unfortunately I couldn't find out why, so I had this solution, which first collects all attributes and then removes them later.

**Pull request merged**